test/integration/services/patient-quick-search.js

```diff
@@ -17,7 +17,9 @@ context('Patient Quick Search', function() {
 
         return fx;
       })
-      .visit();
+      .visit()
+      .navigate('/worklist/owned-by')
```

Comment:
why does this need to navigate or wait at all?

Reply:
it sure doesn't